downloads/swan-lake-release-notes/swan-lake-beta1/RELEASE_NOTE.md

```diff
@@ -38,5 +38,9 @@ If you have not installed jBallerina, then download the [installers](/downloads/
 - 
 
 #### Developer Tools
-- 
+
+##### Debugger
+
+- Added remote debug support with the executable JAR.
+- Added support to debug a single test case with Visual Studio Code.
```

Comment:
```suggestion
- Added support for debugging Ballerina individual tests in Visual Studio Code.
```

Reply:
How about saying: "Added support for debugging single Ballerina tests in Visual Studio Code."?

Reply:
+1

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-dev-website/pull/2197/commits/0b5acefc50bac6542f271cb41574a1402d95fd0d